### ISSUE TITLE
Add support for mega merged mods

### ISSUE THREAD

**Issue by Alex-Au1** [nhok0169/Anime-Game-Remap#14]
For **merged mods**, the script creates the `[CommandListRaidenShogunRemapBlend]` mapping function to all `*RemapBlend.buf` by binding all the those files to the `$swapvar` variable

**eg.**
```ini
[CommandListRaidenShogunRemapBlend]
if $swapvar == 0
   vb1 = ResourceRaidenShogunRemapBlend.0
   ...
else if $swapvar == 1
   vb1 = ResourceRaidenShogunRemapBlend.1
   ...
else if $swapvar == 2
   vb1 = ResourceRaidenShogunRemapBlend.2
   ...
...
...
endif
```

For "**_mega merged mods_**" (mods with **multiple swap keys**), need to follow the same _if...else_ template from `[CommandListRaidenShogunBlend]` instead of creating the conditions ourselves

**eg.**
```ini
[CommandListRaidenShogunRemapBlend]
if $swapvar == 0 && $swapvarn == 0
   vb1 = ResourceRaidenShogunRemapBlend.0
   ...
else if $swapvar == 1 && $swapvarn == 0
   vb1 = ResourceRaidenShogunRemapBlend.1
   ...
else if $swapvar == 2 && $swapvarn == 0
   vb1 = ResourceRaidenShogunRemapBlend.2
   ... 
else if $swapvarn == 1
   vb1 = ResourceRaidenShogunRemapBlend.3
   ...
endif
```

**Comment by Alex-Au1** [comment]
Going to use _**Issues**_ as a **backlog** for any incoming **feature/bug requests** (since going to be busy for a while)

**Comment by Alex-Au1** [comment]
Now in development...

**Comment by Alex-Au1** [comment]
**Completed:** https://github.com/nhok0169/Fix-Raiden-Boss/pull/15
